### PR TITLE
fix: Reorder the tab in getting started

### PR DESCRIPTION
**Pull request by gaocegege**
Keep pip in the first tab, then homebrew, pipx, and installation script.

Signed-off-by: Ce Gao <cegao@tensorchord.ai>

### CODE REVIEW

File: docs-zh/guide/getting-started.md [hunk 1]
```diff
@@ -119,12 +88,12 @@ def build():
 ### 安装和初始化 `envd`
 
 <tabs>
-<tab name="安装脚本">
+<tab name="pip">
 
-在终端中运行以下命令即可安装最新版本的 `envd`:
+`envd` 也可以用 `pip` 来安装（只支持 Python3）。
```
Comment (kemingy) [inline docs-zh/guide/getting-started.md:93]:
```suggestion
`envd` 也可以用 `pip` 来安装。
```

File: docs/guide/getting-started.md [hunk 2]
```diff
@@ -118,12 +86,12 @@ We’re focused on helping data scientists and teams that develop AI/ML models.
 ### Install and bootstrap `envd`
 
 <tabs>
-<tab name="Install Script">
+<tab name="pip">
 
-Run the following command in your terminal to install the latest release of `envd`:
+`envd` can be installed with `pip` (only support Python3).
```
Comment (kemingy) [inline docs/guide/getting-started.md:91]:
```suggestion
`envd` can be installed with `pip`.
```

File: docs/blog/docker-nix-2022.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+author: 'Ce Gao'
+avatar: 'https://www.github.com/gaocegege.png'
+github: 'https://github.com/gaocegege'
+twitter: 'https://twitter.com/gaocegege'
+introduction: 'Do cool things that matter | Co-founder @tensorchord | Co-chair @kubeflow | ex-@Tencent, @bytedance, @caicloud'
+---
```
Comment (kemingy) [inline docs/blog/docker-nix-2022.md:7]: leak?

Comment (gaocegege) [reply docs/blog/docker-nix-2022.md:7]: Sorry for that.

**Comment by gaocegege** [comment]
Comments are addressed, please take a look.
Thanks for your time :smile:

**Review by kemingy — APPROVED** [review]
/lgtm

**Pull request merged**